_infra/helm/ras-rm-sample/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: ras-rm-sample
```

Comment:
This would probably need to be specific, perhaps csv-worker.
There will be an additional help change for sample-api and sample-file-uploader. They cant all have the same chart name and deployment labels. Deployments will not be able to match the pods. Although we discuss more in tech session.

Reply:
One to discuss I think